test/spec/modules/reconciliationRtdProvider_spec.js

```diff
@@ -0,0 +1,63 @@
+import { reconciliationSubmodule, track } from 'modules/reconciliationRtdProvider.js';
```

Comment:
Please add additional unit tests. Coverage is currently at ~35%

Reply:
Fixed. Coverage is ~92%